tests/foreman/cli/test_satellitesync.py

```diff
@@ -170,42 +301,114 @@ def test_positive_export_rh_product(self, class_export_directory, module_org):
+        assert len(result.stdout) == 0
+        # Export content view
+        ContentExport.completeVersion({'id': cvv['id'], 'organization-id': module_org.id})
+        # Verify export directory is not empty and rh content is present
+        result = ssh.command(f'find {cv_export_dir} -name "*.json"')
+        assert result.return_code == 0
+        assert len(result.stdout) > 0
+        result = ssh.command(f'find {cv_export_dir} -name "*.tar.gz"')
+        assert result.return_code == 0
+        assert len(result.stdout) > 0
+        # test cleanup: remove org dir in export dir
+        result = ssh.command(f'rm -rf {cv_export_dir}{module_org.name}')
         assert len(result.stdout) == 0
 
-        # Synchronize the repository
-        Repository.synchronize({'id': repo['id']})
+    @pytest.mark.skip_if_not_set('fake_manifest')
+    @pytest.mark.tier3
+    @pytest.mark.upgrade
+    def test_positive_complete_library_rh_repo(self):
+        """Export RedHat repo via complete library
 
-        # Export the repository once again
-        Repository.export({'id': repo['id']})
+        :id: ffae18bf-6536-4f11-8002-7bf1568bf7f1
 
-        # Verify RPMs were successfully exported
-        result = ssh.command(f'find {repo_export_dir} -name "*.rpm"')
+        :expectedresults: Repository was successfully exported, exported files are
+            present on satellite machine
+
+        :CaseLevel: System
+        """
+        # Enable RH repository
+        my_org = make_org()
+        cv_name = gen_string('alpha')
+        cv_export_dir = '/var/lib/pulp/exports/'
+        with manifests.clone() as manifest:
+            ssh.upload_file(manifest.content, manifest.filename)
+        Subscription.upload({'file': manifest.filename, 'organization-id': my_org['id']})
+        RepositorySet.enable(
+            {
+                'basearch': 'x86_64',
+                'name': REPOSET['rhva6'],
+                'organization-id': my_org['id'],
+                'product': PRDS['rhel'],
+                'releasever': '6Server',
+            }
+        )
+        repo = Repository.info(
+            {
+                'name': REPOS['rhva6']['name'],
+                'organization-id': my_org['id'],
+                'product': PRDS['rhel'],
+            }
+        )
+        # Update the download policy to 'immediate' and sync
+        Repository.update({'download-policy': 'immediate', 'id': repo['id']})
+        Repository.synchronize({'id': repo['id']}, timeout=1000)
+        # Create cv and publish
+        cv = make_content_view({'name': cv_name, 'organization-id': my_org['id']})
+        ContentView.add_repository(
+            {
+                'id': cv['id'],
+                'organization-id': my_org['id'],
+                'repository-id': repo['id'],
+            }
+        )
+        ContentView.publish({'id': cv['id']})
+        # Verify export directory is empty
+        result = ssh.command(f'find {cv_export_dir} -name "*.json"')
```

Comment:
These 4/5 lines for asserting against the `find`s are repeated several times. Consider defining a function in the test module that takes the `default_sat` instance and directory as arguments and makes the assertions against the output and status.